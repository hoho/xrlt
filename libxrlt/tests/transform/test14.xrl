<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">

    <xrl:function name="ololo">
        <hello>world</hello>
    </xrl:function>

    <xrl:import href="test14i.xrl" />

    <xrl:response>
        <xrl:apply name="ololo" />|<xrl:apply name="ololo2" />

    </xrl:response>

</xrl:requestsheet>
