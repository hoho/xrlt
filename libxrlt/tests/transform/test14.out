XRLT_STATUS_CHUNK
chunk: hello
chunk: |
chunk: yo
XRLT_STATUS_DONE
